### PR TITLE
fix: any variable can contains redefines clause

### PR DESCRIPTION
**Pull request by grianbrcom**

### CODE REVIEW

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/VariableDefinitionUtil.java [hunk 1]
```diff
@@ -383,11 +389,12 @@ private VariableWithLevelNode getVariableForConditional(VariableDefinitionNode d
     return null;
   }
 
-  // TODO: 5. 66 and 88 cannot be redefined
-  private List<SyntaxError> handleRedefines(VariableNode variableNode,
-                                                   VariableDefinitionNode definitionNode) {
-    if (definitionNode.doesntHaveRedefines()) return ImmutableList.of();
-    List<SyntaxError> errors = new ArrayList<>();
+  // TODO: 66 and 88 cannot be redefined
```
Comment (temanbrcom) [inline server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/VariableDefinitionUtil.java:392]: Do we still need this TODO?

**Review by temanbrcom — APPROVED** [review]
🥼

**Review by asatklichov — APPROVED** [review]
🌊

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/VariableDefinitionUtil.java (hunk 1 above)
Comment (grianbrcom) [reply server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/VariableDefinitionUtil.java:392]: Another TODO was added.

**Pull request merged**